client/components/TestRun/index.jsx

```diff
@@ -447,8 +431,23 @@ const TestRun = () => {
             !rendererState ||
             !testResult.scenarioResults ||
             rendererState.commands.length !== testResult.scenarioResults.length
-        )
-            return testResult;
+        ) {
+            // Mapping unexpected behaviors to expected TestRenderer downstream format
+            const scenarioResults = testResult.scenarioResults.map(
+                scenarioResult => {
+                    return {
+                        ...scenarioResult,
+                        unexpectedBehaviors: scenarioResult.unexpectedBehaviors
+                            ? scenarioResult.unexpectedBehaviors.map(
+                                  behavior => behavior.id
+                              )
+                            : scenarioResult.unexpectedBehaviors
```

Comment:
Makes sense! This could be simplified slightly to `scenarioResult.unexpectedBehaviors ?? scenarioResult.unexpectedBehaviors.map(behavior => behavior.id)`